docs/current/guides/324301-argo-workflows.md

```diff
@@ -0,0 +1,110 @@
+---
+slug: /324301/argo-workflows
+displayed_sidebar: "current"
+category: "guides"
+tags: ["python", "go", "nodejs", "argo", "kubernetes"]
+authors: ["Kyle Penfound"]
+date: "2023-08-22"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Use Dagger with Argo Workflows
+
+## Introduction
+
+This guide explains how to run Dagger pipelines in Argo Workflows. According to their [site](https://argoproj.github.io/argo-workflows/), Argo Workflows is an open source container-native workflow engine for orchestrating parallel jobs on Kubernetes.
+
```

Comment:
## Requirements

This guide assumes that you have a basic understanding of Kubernetes and Argo Workflows.

Reply:
Are there are prerequisites? If yes we should add them here following the format in https://github.com/dagger/dagger/blob/main/docs/current/guides/183109-aws-codebuild-codepipeline.md?plain=1#L23C1-L34C1

Reply:
No prerequisites. And personally I think stating that you need understanding of kubernetes and argo workflows for an argo guide isn't worth noting

Reply:
I added this section primarily to have consistency with other guides.

Reply:
ok, added